setup.py

```diff
@@ -22,9 +22,9 @@
     scripts=[],
     author="Anaconda, Inc.",
     author_email="numba-users@continuum.io",
-    url="https://github.com/numba/numba_scipy",
-    download_url="https://github.com/numba/numba_scipy",
-    packages=['numba_scipy'],
+    url="https://github.com/numba/numba-scipy",
+    download_url="https://github.com/numba/numba-scipy",
+    packages=['numba-scipy'],
```

Comment:
I think this one has to stay `numba_scipy` since it is the name of the top level directory.

Reply:
Well spotted. Fixed.

Reply:
Alternatively, `dask-kubernetes` seems to use the `find_packages` method:

https://github.com/dask/dask-kubernetes/blob/master/setup.py